sdk/core/core-tracing/Changelog.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.0-preview.2 9th September 2019
+
+Adds support for retrieving span context. (PR #4712)
```

Comment:
```suggestion
Updated the `OpenCensusSpanPlugin` to support for retrieving span context. (PR [#4712](https://github.com/Azure/azure-sdk-for-js/pull/4712))
```

Reply:
Can you include the link to the trace context spec here? The one @bterlson shared with us as to what the trace context should look like